TODO

```diff
@@ -32,6 +32,10 @@ Janitorial Clean-ups:
 
 Features:
 
+* add a percentage syntax for TimeoutStopSec=, e.g. TimeoutStopSec=150%, and
+  then use that for the setting used in user@.service. It should be understood
+  relative to the configured default value.
```

Comment:
by default you mean the DefaultTimeoutSec value, not the default for TimeoutSec?